languages/javascript/exercises/concept/array-transformations/.docs/after.md

```diff
@@ -0,0 +1 @@
+Remember, anytime you're about to write a for loop to transform an array there is a good chance Array has already has you covered with a built-in.
```

Comment:
The `after.md` document should contain all the information that is in the `introduction.md` file. Part of the reason for this is that we'll make it easy for students to revisit the `after.md` document later, so it will be like a reference document for the involved concepts. The nice thing about the `after.md` document is that you can use it to expand upon things. For a list of things that you could include in the `after.md` document that you wouldn't include in the `introduction.md`, see https://github.com/exercism/v3/blob/master/docs/concept-exercises.md#docsaftermd.

Reply:
Yeah, this was more of a 'placeholder'... I'll watch the video again and read your link and then circle back to this.

Reply:
Okay great! We've been toying around with the idea of putting the literal text `TODO` in documents that need updating later. Something to consider doing if you'd like reviewers to ignore files when reviewing.